### PR TITLE
Add domain verification for runshaw.ac.uk

### PR DESCRIPTION
**Pull request by Dragon863**
# Updating `runshaw.hackclub.com`

## Description

Hi again! Nest requires me to add a TXT record for domain verification before I can set up the reverse proxy to work correctly. Thank you!

### CODE REVIEW

**Comment by alx-alexpark** [comment]
Hii, could you address the failing octodns check?

**Comment by Dragon863** [comment]
Will do.

**Comment by Dragon863** [comment]
It's probably obvious, but I can't figure out what isn't valid?

**Comment by Dragon863** [comment]
Should be fixed, I tried a different approach

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2380,8 +2380,12 @@ ru:
...
 runshaw:
   - ttl: 600
-    type: CNAME
-    value: dragon863.hackclub.app.
+    type: ALIAS
+    value: hackclub.app.
+  - ttl: 600
+    type: TXT
+    values:
+      - domain-verification=dragon863
```
Comment (Muirrum) [inline hackclub.com.yaml:2388]: Please put this at the top of the file like the README says

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (Dragon863) [reply hackclub.com.yaml:2388]: Sorry, I'm not sure I understand which part I should be moving? If you're referring to the txt record I don't think it should be under the vercel section as I'm trying to verify with nest as shown in [the nest wiki](https://guides.hackclub.app/index.php/Subdomains_and_Custom_Domains)

Comment (Muirrum) [reply hackclub.com.yaml:2388]: this is what I get for reviewing on my phone 🤦‍♀️

**Review by Muirrum — APPROVED** [review]
lgtm

**Comment by Dragon863** [comment]
Thanks! Not sure why the deploy action failed though? All the checks passed

**Pull request merged**